<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDialog</class>
 <widget class="QDialog" name="UpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>升级系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/picture/picture/003.png</normaloff>:/picture/picture/003.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_browser">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>192</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择升级包</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>选择升级系统IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_IP">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_upload">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>276</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>升级</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_file">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_upload">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>276</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>当前升级系统IP为：</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::CurrentChanged|QAbstractItemView::DoubleClicked|QAbstractItemView::SelectedClicked</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
